src/common/utility.cpp

```diff
@@ -23,47 +23,47 @@
 #include "version.h"
 
 // Note:  This file must compile without QtGui
+#include <QCollator>
 #include <QCoreApplication>
-#include <QSettings>
-#include <QTextStream>
+#include <QDateTime>
 #include <QDir>
 #include <QFile>
-#include <QUrl>
-#include <QProcess>
 #include <QObject>
-#include <QThread>
-#include <QDateTime>
-#include <QSysInfo>
+#include <QProcess>
+#include <QRandomGenerator>
+#include <QSettings>
 #include <QStandardPaths>
-#include <QCollator>
 #include <QSysInfo>
-
+#include <QTextStream>
+#include <QThread>
+#include <QUrl>
 
 #ifdef Q_OS_UNIX
 #include <sys/statvfs.h>
 #include <sys/types.h>
 #include <unistd.h>
 #endif
 
+#include <cstring>
 #include <math.h>
 #include <stdarg.h>
-#include <cstring>
 
 namespace OCC {
 
 Q_LOGGING_CATEGORY(lcUtility, "sync.utility")
 
 bool Utility::writeRandomFile(const QString &fname, int size)
```

Comment:
Hm can you move that function to https://github.com/owncloud/client/tree/master/test/testutils ?